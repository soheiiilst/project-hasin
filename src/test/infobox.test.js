import React from 'react';
import { shallow } from 'enzyme';
import InfoBox from '../component/infobox/infobox.component';

it('expect to render InfoBox component', () => {
  expect(shallow(<InfoBox />)).toMatchSnapshot();
})
