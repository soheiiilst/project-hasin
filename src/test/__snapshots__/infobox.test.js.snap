// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render InfoBox component 1`] = `ShallowWrapper {}`;
